src/extension.ts

```diff
@@ -81,7 +82,7 @@ export async function activateInternal(ctx: vscode.ExtensionContext, perfStats:
     await callWithTelemetryAndErrorHandling('docker.activate', async (activateContext: IActionContext) => {
         activateContext.telemetry.properties.isActivationEvent = 'true';
         activateContext.telemetry.measurements.mainFileLoad = (perfStats.loadEndTime - perfStats.loadStartTime) / 1000;
-        activateContext.telemetry.properties.dockerInstallationID = await getDockerInstallationID();
+        activateContext.telemetry.properties.dockerInstallationIDHash = await getDockerInstallationIDHash();
```

Comment:
Changing the name is needed because the old data's classification cannot be changed.